stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/reactions/user/internal/UserReactionItem.kt

```diff
@@ -1,12 +1,18 @@
 package io.getstream.chat.android.ui.message.list.reactions.user.internal
 
-import androidx.annotation.DrawableRes
 import io.getstream.chat.android.client.models.Reaction
 import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.ui.SupportedReactions
 
 internal data class UserReactionItem(
     val user: User,
     val reaction: Reaction,
     val isMine: Boolean,
-    @DrawableRes val iconDrawableRes: Int,
-)
+    private val reactionDrawable: SupportedReactions.ReactionDrawable,
+) {
+    val drawable = if (isMine) {
```

Comment:
This snippet is duplicated, maybe better to have:


```
public class ReactionDrawable(public val inactiveDrawable: Drawable?, public val activeDrawable: Drawable?) {
    getDrawable(isActive: Boolean): Drawable? {
        return if (isActive) {
             activeDrawable
        } else {
             inactiveDrawable
        }
    }
}
```

Reply:
Fixed